src/modules/launcher/Plugins/Community.PowerToys.Run.Plugin.ValueGenerator/Main.cs

```diff
@@ -150,13 +196,42 @@ private Result GetResult(IComputeRequest request)
             };
         }
 
+        private List<Result> GetSuggestionFuzzyResults(Query query, List<Result> results)
+        {
+            foreach (var generatorData in QueryHelper.GeneratorDataList)
+            {
+                var matchScore = StringMatcher.FuzzySearch(query.Search, generatorData.Keyword).Score;
```

Comment:
```suggestion
                var matchScore = StringMatcher.FuzzySearch(query.Search.Trim(), generatorData.Keyword).Score;
```

Is always a good idea.